experiments/clang_interface/Makefile

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
Why Make, rather than CMake?

Reply:
Because this is *very* incomplete.  This is just about enough to test it on my machine, but it's not sufficiently complete for anything other than copying and pasting into other bits of the system.  I doubt actually running it is useful to anyone.